master.jinja2

```diff
@@ -1,10 +1,38 @@
+{# timeouts #}
+{% if lxc_deploy_timeout is undefined %}{% set lxc_deploy_timeout = 15 %}{% endif %}
+{% if lxc_boot_timeout is undefined %}{% set lxc_boot_timeout = 5 %}{% endif %}
+{% if lxc_install_fastboot_timeout is undefined %}{% set lxc_install_fastboot_timeout = 10 %}{% endif %}
+{% if target_deploy_timeout is undefined %}{% set target_deploy_timeout = 40 %}{% endif %}
+{% if target_boot_timeout is undefined %}{% set target_boot_timeout = 15 %}{% endif %}
+{% if test_timeout is undefined %}{% set test_timeout = 60 %}{% endif %}
+{% set job_timeout = lxc_deploy_timeout + lxc_boot_timeout + lxc_install_fastboot_timeout + target_deploy_timeout + target_boot_timeout + test_timeout %}
+
 {% block global_settings %}
+timeouts:
+  job:
+    minutes: {% if job_timeout is defined %}{{ job_timeout }}{% else %}20{% endif %}
+  connection:
+    minutes: 2
+
+{% if use_context is defined and use_context == true %}
+context:
+{% block context %}
+{% endblock context %}
+{% endif %}
+
+{% block settings %}
+{% endblock settings %}
 {% endblock global_settings %}
 
+
 device_type: {% block device_type %}{% endblock %}
 job_name: {% block job_name %}{% endblock %}
-priority: {% block priority %}{% endblock %}
-visibility: {% block visibility %}{% endblock %}
+priority: {% block priority %}{{LAVA_JOB_PRIORITY}}{% endblock priority %}
+visibility: {% block visibility %}{% if LAVA_JOB_VISIBILITY is undefined %}public{% else %}{{LAVA_JOB_VISIBILITY}}{% endif %}{% endblock visibility %}
+
+{# variables #}
+{% if rootfs is undefined %}{% set rootfs = true %}{% endif %}
+{% if rootfs_label is undefined %}{% set rootfs_label = 'rootfs' %}{% endif %}
```

Comment:
IIRC these are db410c specific. Is there a reason to set them in master template?

Reply:
That was defined in lkft.jinja2...